tests/libvmi/vmi.go

```diff
@@ -92,6 +92,12 @@ func WithRng() Option {
 	}
 }
 
+func WithNoRNG() Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Devices.Rng = nil
+	}
+}
```

Comment:
We already have `WithRng()`!
I don't think we need this new one.
Maybe extend the `WithRng()` functionality like this?
```
func WithRng(rng *v1.Rng) Option {
	return func(vmi *v1.VirtualMachineInstance) {
		vmi.Spec.Domain.Devices.Rng = rng
	}
}
```

Reply:
I tend to disagree. The `WithRng` name clearly states that something is present, where the other explicitly states that something is not present. We could argue that the name is not well chosen, so how about this:

```suggestion
func WithoutRNG() Option {
	return func(vmi *v1.VirtualMachineInstance) {
		vmi.Spec.Domain.Devices.Rng = nil
	}
}
```

WDYT?

Reply:
Side note: Rng is an acronym and should be written `RNG` as it states from Random Number Generator. But since the field is named like this I stuck to the field name.

Reply:
I think the latter is the best solution :)

Reply:
Done